distutils/cygwinccompiler.py

```diff
@@ -124,33 +123,14 @@ def __init__(self, verbose=0, dry_run=0, force=0):
         self.cc = os.environ.get('CC', 'gcc')
         self.cxx = os.environ.get('CXX', 'g++')
 
-        if ('gcc' in self.cc): # Start gcc workaround
-            self.gcc_version, self.ld_version, self.dllwrap_version = \
```

Comment:
To clarify, this patch removes `self.ld_version` and `self.dllwrap_version` completely. If this is a potential problem re backwards compat (I personally doubt it) then we can try to set them to dummy values too.

Reply:
>  If this is a potential problem re backwards compat (I personally doubt it)

I appreciate the sentiment. Let's try without backwards compat and then revise if needed.